dangerfile.ts

```diff
@@ -93,6 +112,7 @@ export default async () => {
       'There are no updates provided to CHANGELOG. Ensure there are no publicly visible changes introduced by this PR.',
     )
   } else {
+    await checkChangelogFormat()
```

Comment:
```js
hasImproperlyFormattedLinesInChangelog().then(lines => {
  lines.forEach(line => fail(`Invalid entry format in ${CHANGELOG_FILE}: >${line.content}<`))
})
```

Can we be consistent and keep all warnings and failures inside default function?